app/code/Transoft/Blog/Api/Data/ModelSearchResultsInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
Why do we need this. These methods already defined in \Magento\Framework\Api\SearchResultsInterface

Reply:
removed